mappings/com/mojang/blaze3d/platform/GlStateManager.mapping

```diff
@@ -470,6 +475,7 @@ CLASS net/minecraft/class_4493 com/mojang/blaze3d/platform/GlStateManager
 	METHOD method_23282 setupOutline ()V
 	METHOD method_23283 teardownOutline ()V
 	CLASS class_1010 FBOMode
+		COMMENT
```

Comment:
Empty comment ?

Reply:
Uh... what? I added a comment on there by accident and intended to remove it.

Doesn't clearing the box delete the comment? #bug